src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -207,28 +231,24 @@ const styles = StyleSheet.create({
     marginLeft: 12,
     flexDirection: 'column',
     flex: 1
-    // justifyContent: 'space-between',
   },
   wrapTextProfileTopic: {
     flexDirection: 'column',
     flex: 1
-    // justifyContent: 'space-between',
   },
   textProfileUsername: {
     fontFamily: fonts.inter[500],
     fontWeight: 'bold',
     fontSize: 14,
     color: COLORS.black,
     lineHeight: 16.94
-    // backgroundColor: 'red',
   },
   textProfileFullName: {
     fontFamily: fonts.inter[400],
     fontSize: 12,
     color: COLORS.blackgrey,
     flexWrap: 'wrap',
     lineHeight: 18,
-    // backgroundColor: 'green',
     marginTop: 4
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Comments have been removed from the styles. If these comments were important for understanding the code, consider adding them back.



<!-- This is an auto-generated comment by OSS CodeRabbit -->